docs/source/en/model_doc/xlm-prophetnet.mdx

```diff
@@ -55,14 +55,93 @@ The Authors' code can be found [here](https://github.com/microsoft/ProphetNet).
 
 [[autodoc]] XLMProphetNetEncoder
 
+```python
```

Comment:
You have examples in each docstring, so you really don't need to add those here or there will be present twice in the doc (basically there should be no modifications to the doc file).

Reply:
Ah I see that now. Thanks - pushed the changes.